lib/server/service.tsx

```diff
@@ -458,3 +458,30 @@ export async function isTenantPublic(slug: string) {
   const tenant = await getTenantBySlug(slug);
   return tenant?.isPublic ?? false;
 }
+
+export async function getAuthenticatedMembersByTenantId(tenantId: string): Promise<Member[]> {
+  return union(
+    db
+      .select({
+        id: schema.profiles.id,
+        email: schema.users.email,
+        name: schema.users.name,
+        type: sql<MemberType>`'profile'`.as("type"),
+        role: schema.profiles.role,
+      })
+      .from(schema.profiles)
+      .innerJoin(schema.users, eq(schema.profiles.userId, schema.users.id))
+      .innerJoin(schema.tenants, eq(schema.tenants.id, tenantId))
+      .where(and(eq(schema.profiles.tenantId, tenantId), eq(schema.users.type, "authenticated"))),
```

Comment:
I think we should just have an is_anonymous field.  It's something that I'll be working through in my PR.  You'll see how it ties into the auth system.